app/src/main/java/com/example/awesomelanguagelearning/home/ui/views/home/CircularProgressbarWithTitle.kt

```diff
@@ -0,0 +1,91 @@
+package com.example.awesomelanguagelearning.home.ui.views.home
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.example.awesomelanguagelearning.R
+import com.example.awesomelanguagelearning.core.ui.theme.AppTheme
+import com.example.awesomelanguagelearning.core.ui.views.TextTitle
+
+@Composable
+fun CircularProgressbarWithTitle(
+    number: Int = 15,
+    maxNumber: Int = 30,
```

Comment:
@igel911 let's create constants to avoid magic numbers

Reply:
Actually, I don't need these default values. Moved them to previews